app/models/state_file_az_intake.rb

```diff
@@ -86,7 +85,7 @@
 #  index_state_file_az_intakes_on_spouse_state_id_id   (spouse_state_id_id)
 #
 class StateFileAzIntake < StateFileBaseIntake
-  self.ignored_columns = %w[charitable_cash charitable_noncash household_excise_credit_claimed_amt tribal_wages armed_forces_wages]
+  self.ignored_columns = %w[charitable_cash charitable_noncash household_excise_credit_claimed_amt tribal_wages armed_forces_wages bank_name df_data_import_failed]
```

Comment:
Why did you add `df_data_import_failed` as well in this list of ignored columns?

Reply:
the one we actually use is called `:df_data_import_failed_at`. i tried to figure out the provenance of `:df_data_import_failed` but couldn't really get anywhere with it? [it seems like there was a move to replace `failed_at`](https://github.com/codeforamerica/vita-min/commit/0a2e8640995bb181cc7a5b2cc75524f574bd4a90). but i'm not sure how `:df_data_import_failed` was introduced...

Reply:
this is a typo... 😓